.jenkins/nightly.groovy

```diff
@@ -1,4 +1,8 @@
 pipeline {
+    options {
+        disableConcurrentBuilds(abortPrevious: true)
```

Comment:
Why did you set this option?

Reply:
I just copied it from the other file and couldn't quite decide to keep it or not. As I said in the initial comment, it probably isn't really relevant here. I'm happy to remove if you prefer.

Reply:
Remove it